edx_name_affirmation/statuses.py

```diff
@@ -31,3 +31,35 @@ class VerifiedNameStatus(str, Enum):
     SUBMITTED = "submitted"
     APPROVED = "approved"
     DENIED = "denied"
+
+    @classmethod
+    def translate_idv_status(cls, idv_status):
```

Comment:
translate feels like not quite the right concept for trigger a state transition but only sometimes

maybe trigger_state_change_from_idv or similar

Reply:
+1